tests/code/test_calculatingRecurringEventFrequency.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from app.logic.adminNewEvent import calculateRecurringEventFrequency
+
+@pytest.mark.integration
+def test_calculateRecurringEventFrequency():

```

Comment:
This tests that the function runs, but you don't actually test if it works. And empty function would pass these tests.

Reply:
This test was for a function that is not being used in this PR. The file has been removed to avoid confusion. The other PR for recurring events has the propper tests.